scripts/sas-trading.js

```diff
@@ -262,19 +282,19 @@ class SasTradingGoodData {
      */
     static createGood(good) {
         if (!good.name || !good.city) {
-            SasTrading.log(false, 'good must have at least name and city properties', good)
+            SasTrading.error('good must have at least name and city properties', good)
             return
         }
         if (!good.id) {
             good.id = this.goodId(good.name, good.city)
         } else if (good.id !== this.goodId(good.name, good.city)) {
-            SasTrading.log(false, 'good.id "', good.id, '" does not match expected id:', this.goodId(good.name, good.city))
+            SasTrading.error(false, 'good.id "', good.id, '" does not match expected id:', this.goodId(good.name, good.city))
             return
         }
 
         const goods = this.allGoods
         if (goods.hasOwnProperty(good.id)) {
-            SasTrading.log(false, 'good already exists:', good.id)
+            SasTrading.warn('good already exists:', good.id)
             // TODO: Application Warning? Error?
```

Comment:
This should probably get a notification since it doesn't have one in the form application

Reply:
Done